### ISSUE TITLE
Estacionamento Gratuito

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments-new#271]
### Post de Referência:

https://www.vish.blog.br/imagens/gifs/estacionamento-gratuito/

**Comment by one-eyed-willy** [comment]
A genialidade nem foi a ideia das pernas falsas em si e sim jogar as ferramentas próximas a elas pra ninguém desconfiar.

**Comment by angellisy** [comment]
> [Willy Caolho](/perfil/?user=one-eyed-willy) disse [segunda-feira, 10 de outubro de 2022](https://www.vish.blog.br/imagens/gifs/estacionamento-gratuito/#comment_1273422341):

>A genialidade nem foi a ideia das pernas falsas em si e sim jogar as ferramentas próximas a elas pra ninguém desconfiar.

Verdade. Sem esse pequeno detalhe, qualquer um poderia cutucar as pernas e descobrir a façanha.

**Comment by brunoaugusto** [comment]
![Paulinho Gogó - "Quem não tem dinheiro conta estória"](https://i.imgur.com/SqWXUhl.png)
